torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6219,6 +6219,7 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            ),
            sample_inputs_func=sample_inputs_xlog1py),
     OpInfo('logsumexp',
+           aliases=('special.logsumexp',),
```

Comment:
`alias_remapping` test fails due to the way script is generated in `remapping_alias` vs `consistency_jit` tests.

```python
import torch

t = torch.zeros((2, 3, 3))
torch.special.logsumexp(t, (0,), False)


t = torch.zeros((2, 3, 3))
torch.special.logsumexp(t, [0, ], False)

variant = torch.special.logsumexp
t0 = t
s0 = (0,)
s1 = False

# As generated by `consistency_jit` test
# Works
script = """
def _fn(t0):
    return variant(t0, (0,), True)
"""

torch.jit.CompilationUnit(script)._fn

# As generated by `alias_remapping` test
# Doesn't work due to the type being generated as `tuple`.
script = """
def _fn(t0, s0: tuple, s1: bool):
    return variant(t0, s0, s1)
"""

torch.jit.CompilationUnit(script)._fn

```